src/src/targets.h

```diff
@@ -194,8 +194,8 @@ Credit to Jacob Walser (jaxxzer) for the pinout!!!
 #define GPIO_PIN_DEBUG_TX    PA9 // confirmed
 
 
-#define BUFFER_OE               PA5  //CONFIRMED
-#define BUFFER_OE_ACTIVE HIGH
+#define GPIO_PIN_RC_INVERT      PA5  //CONFIRMED
```

Comment:
this naming is not very suitable. That pin is also used to control buffer directions (originally).

Reply:
So what do you suggest then? `GPIO_PIN_RCSIGNAL_TX` is already used to name the TX GPIO, what's a name for a secondary pin that enabled TX? How about `GPIO_PIN_RC_TX_EN` and `GPIO_PIN_RC_TX_EN_INVERT`?

Reply:
myabe `GPIO_PIN_RCSIGNAL_RX_BUFFER` and `GPIO_PIN_RCSIGNAL_TX_BUFFER`.
Those follow the current RCSIGNAL_PIN naming :)
and `_INVERT` suffix...
@AlessandroAU any comment?